app/styled/WrapPlot.js

```diff
@@ -1,8 +1,11 @@
 import styled from 'styled-components';
 
 export default styled.div`
+  width: 100%;
+  max-width: 300px;
   @media (min-width: ${({ theme }) => theme.breakpointsMin.large}) {
     padding-right: 50px;
     padding-left: 50px;
+    max-width: 500px;
```

Comment:
@iainkirkpatrick just a couple of thoughts here

- this also affects the ESR charts, not only CPR as requested, I believe it would be beneficial to use the available space for ESR
- further my preference would still be to add padding inside the chart (the min/max dates offset from the extremes) than outside the chart, by reducing the width. I believe this would also address the issue @annemariebrook described ("makes it difficult to see if the line is trending up or down.")

Reply:
Current ESR with dates stuck on the extremes
![image](https://user-images.githubusercontent.com/5811956/85151128-b3651f80-b253-11ea-9362-8c0c3b28eba2.png)

Current CPR
![image](https://user-images.githubusercontent.com/5811956/85151230-d55ea200-b253-11ea-8fd1-15c1cfed9907.png)